app/entities/errors.go

```diff
@@ -0,0 +1,5 @@
+package entities
+
+import "errors"
+
+var ErrDuplicateEmail = errors.New("email is already taken")
```

Comment:
It's not entities-specific error, entities themselves should not know about any storages and any requirements for the emails to be unique, etc. Entities just provide models. This is a bad package for this error because it shouldn't be the responsibility of the `entities` package.

Reply:
Is `usecase` a better place for this kind of error?

Reply:
maybe use http/const.go ?

Reply:
The best place - is the package which returns this error.